homework/test/test-list.js

```diff
@@ -2,7 +2,8 @@ describe('Test > ', function() {
   beforeEach(function() {
   });
 
-  it('will test some pure functions', function() {
+  it('will test some pure functions', function () {
+      assert.deepEqual(test.test(1,1), 2);
```

Comment:
Well, you may need to check if you did successfully include the `test.js` and test it here.